src/components/LangBar/LangBar.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+import {withTranslation} from 'react-i18next'
+
+import LangBarItem from './LangBarItem'
+
+class LangBar extends React.Component {
+  languages = [
+    {code: 'en', label: 'English'},
+    {code: 'bg', label: 'Български'}
+  ]
+
+  handleClick = (code) => {
+    this.props.i18n.changeLanguage(code);
+  }
+
+  render() {
+    return (
+      <div>
```

Comment:
A language title could be added: 

Language: English Български 

I know that Language is English, but it will look like section and English is a common language.

Reply:
Nevermind. It's ok now.